src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(value, a) {
```

Comment:
```suggestion
    reset() {
      this.result = 0;

      return this;
    },
    
    add(value, a) {
```

Add an empty line between methods